BinaryObjectScanner/Protection/CrypKey.cs

```diff
@@ -17,16 +23,41 @@ public class CrypKey : IPortableExecutableCheck
             if (sections == null)
                 return null;
 
+            // Get the .data/DATA section strings, if they exist
+            var strs = pex.GetFirstSectionStrings("CODE");
+            if (strs != null)
+            {
+                // Found in "NECRO95.EXE" in IA item "NBECRORV11".
+                // Full string:
+                // *CrypKey Instant 2.0 security i(32 - bit)  *
+                // *Copyright(c) 1996 Kenonic Controls Ltd.  *
+                if (strs.Any(s => s.Contains("CrypKey Instant 2.0 security")))
+                    return "CrypKey Instant 2.0";
+
+                // Generic check to catch unknown CrypKey Instant versions.
+                if (strs.Any(s => s.Contains("CrypKey Instant")))
+                    return "CrypKey Instant (Unknown version - Please report to us on GitHub)";
+
+                // Generic check to catch unknown CrypKey products.
+                if (strs.Any(s => s.Contains("CrypKey")))
```

Comment:
What are the odds that this would over-match?

Reply:
Should be quite low that it would over match something we're not interested in.